perun-base/src/main/java/cz/metacentrum/perun/core/api/Member.java

```diff
@@ -119,6 +122,20 @@ public void setSponsored(boolean sponsored) {
 		this.sponsored = sponsored;
 	}
 
+	public Date getSuspendedTo() {
+		return suspendedTo;
+	}
+
+	public void setSuspendedTo(Date suspendedTo) { this.suspendedTo = suspendedTo; }
+
+	public boolean isSuspended() {
+		if(getSuspendedTo() != null) {
+			return suspendedTo.after(Calendar.getInstance().getTime());
```

Comment:
We should probably use new Java Date API (LocalDateTime.now()) instead of Calendar.

Reply:
Of course, fixed.